omnikinverter/omnikinverter.py

```diff
@@ -31,9 +32,13 @@ class OmnikInverter:
     source_type: str = "javascript"
     request_timeout: float = 10.0
     session: ClientSession | None = None
+    serial_number: int | None = None  # Optional, only for TCP backend
+    tcp_port: int = 8899
 
     _close_session: bool = False
 
+    _socket_mock: None = None
```

Comment:
Is this the correct name, mocking mainly takes place in tests? 🤷🏻

Reply:
https://asynctest.readthedocs.io/en/latest/tutorial.advanced.html#mocking-i-o suggests to pass the test socket mock to `asyncio.open_connection(sock=socket_mock)`, and I'm unsure how to do that in any other way, besides passing it through `device()` and `inverter()` as optional keyword argument?